app/features/notes_generator/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please, the `metadata.json` attributes must follow the same principle and structure as the other tools.